### PR TITLE
Enable creating PREORDERED_LOG

### PR DESCRIPTION
**Pull request by pavelkalinnikov**

### CODE REVIEW

File: storage/testonly/admin_storage_tester.go [hunk 1]
```diff
@@ -120,7 +120,7 @@ var (
 		HashStrategy:       trillian.HashStrategy_RFC6962_SHA256,
 		HashAlgorithm:      spb.DigitallySigned_SHA256,
 		SignatureAlgorithm: spb.DigitallySigned_ECDSA,
-		DisplayName:        "Pavel's Pre-ordered Log",
+		DisplayName:        "Pre-ordered Log",
```
Comment (pavelkalinnikov) [inline storage/testonly/admin_storage_tester.go]: This line was longer than 20 chars, which was against the MySQL schema.

Comment (Martin2112) [reply storage/testonly/admin_storage_tester.go]: We could increase that number, not sure why I made it so short.

Comment (pavelkalinnikov) [reply storage/testonly/admin_storage_tester.go]: Meh, I'm fine not having my name in the code :)

Comment (Martin2112) [reply storage/testonly/admin_storage_tester.go]: This is indeed wise.

**Pull request merged**